gui/builtinViews/fittingView.py

```diff
@@ -339,7 +339,11 @@ def removeItem(self, event):
     def removeModule(self, module):
         sFit = service.Fit.getInstance()
         fit = sFit.getFit(self.activeFitID)
-        populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        try:
+            populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        except ValueError:
+            # This module isn't in our list of modules, don't remove anything. Likely a special snowflake.
+            populate = None
```

Comment:
Instead of this, the user shouldn't see the remove module option for the stance to begin with. Revert this and add that plzkthx